test/services/cognitiveServicesVision/computerVisionAPIClient-tests.js

```diff
@@ -0,0 +1,97 @@
+//
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+
+const should = require('should');
+const util = require('util');
+
+const CognitiveServicesCredentials = require('ms-rest-azure').CognitiveServicesCredentials;
+
+const SuiteBase = require('../../framework/suite-base');
+const ComputerVisionAPIClient = require('../../../lib/services/cognitiveServicesVision/lib/computerVision/computerVisionAPIClient');
+const fs = require('fs');
+
+let requiredEnvironment = [
+  { name: 'AZURE_COMPUTER_VISION_KEY', secure: true }
+];
+
+let testPrefix = 'computervision-tests';
+let suite;
+let client;
+let credentials;
+
+describe('Computer Vision', function () {
+  before(function (done) {
+    suite = new SuiteBase(this, testPrefix, requiredEnvironment);
+    suite.setupSuite(function () {
+      credentials = new CognitiveServicesCredentials(process.env["AZURE_COMPUTER_VISION_KEY"]);
+      client = new ComputerVisionAPIClient(credentials, "westus");
+    });
+    done();
+  });
+
+  after(function (done) {
+    suite.teardownSuite(done);
+  });
+
+  beforeEach(function (done) {
+    suite.setupTest(done);
+  });
+
+  afterEach(function (done) {
+    suite.baseTeardownTest(done);
+  });
+
+  describe('Analyze image.', function () {
+    it('should detect words, "I Love You"', function (done) {
+      let fileStream = fs.createReadStream('test/services/cognitiveServicesVision/i_love_you.JPG');
+      client.recognizeTextInStreamWithHttpOperationResponse(fileStream).then((response) => {
+        let url = response.response.headers["operation-location"];
+        let pollUrl = function () {
```

Comment:
Minor comment: try using the `() => {}`  syntax instead of `function () {}` for anonymous/inline functions everywhere.
I see you using this below.